ztools/http/client_test.go

```diff
@@ -7,14 +7,14 @@
 package http_test
 
 import (
-	"crypto/tls"
 	"errors"
 	"fmt"
+	. "github.com/zmap/zgrab/ztools/http"
```

Comment:
yes - this is so that we can simply refer to http package objects directly, `Request` instead of `http.Request` everywhere. akin to Python's `from foo import *`